packages/create-redwood-app/template/api/src/lib/db.ts

```diff
@@ -0,0 +1,78 @@
+// See https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/constructor
+// for options.
+
+import { PrismaClient, Prisma } from '@prisma/client'
+import { logger } from './logger'
+
+// To help you identify particularly slow parameter values,
+// For example, Heroku outputs the slowest queries (that take 2 seconds or more)
+const SLOW_QUERY_THRESHOLD = 2 * 1000 // 2 seconds
+
+/*
+ * Determines the type and level of logging.
+ *
+ * See: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#log
+ */
+const logLevels: Prisma.LogLevel[] = ['info', 'warn', 'error'] // add 'query' in dev?
+
+/*
+ * Generates the Prisma Log Definitions from log levels
+ *
+ * @return Prisma.LogDefinition[]
+ */
+const prismaLoggerOptions = (
+  logLevels: Prisma.LogLevel[]
+): Prisma.LogDefinition[] => {
+  return logLevels?.map((level) => {
+    return { emit: 'event', level } as Prisma.LogDefinition
+  })
+}
+
+/*
+ * Generates the Prisma Log Definitions from log levels
+ *
+ * @return PrismaClient
+ */
+const createPrismaClient = (logLevels: Prisma.LogLevel[]) => {
+  const client = new PrismaClient({
+    log: prismaLoggerOptions(logLevels),
```

Comment:
And just hard code the levels here

Reply:
My reasoning was:

1. Might be able to infer log level from logger
2. Default base on dev or prod and thus construct them 
3. Developer just has to set warn, info, error and not the emit type (event or stdout or get it wrong or know the Prisma emit types). 

I did think that if the `logger.level === 'silent` then this could auto set to `[]` and thus disable Prisma logging.

Reply:
For example:

`const logLevels: Prisma.LogLevel[] = logger.level === 'silent' ? [] : ['info', 'warn', 'error']`